tests/estimators/regression/test_blackbox.py

```diff
@@ -0,0 +1,114 @@
+# MIT License
```

Comment:
I think to run these tests the file `run_tests.sh` needs to be updated with
```python
pytest --cov-report=xml --cov=art --cov-append  -q -vv tests/estimators/regression/ --framework=$framework  --durations=0
    if [[ $? -ne 0 ]]; then exit_code=1; echo "Failed estimators/regression tests"; fi
```
starting in line 70.